src/core/analytics/analyticsreporter.js

```diff
@@ -67,8 +74,19 @@ export default class AnalyticsReporter {
     this._globalOptions.queryId = queryId;
   }
 
+  /**
+   * Opt in or out of analytics click events
+   * @param {boolean} bool
+   */
+  setAnalyticsOptIn (bool) {
```

Comment:
According to the spec, we'll need an `ANSWERS.setAnalyticsOptIn` method as well. This means we will need to make a `setAnalyticsOptIn` method available in the `answers-umd` class. It can be a simple pass-through to this.